projects/optic/src/commands/diff/generate-rule-runner.ts

```diff
@@ -11,6 +11,12 @@ import { loadCliConfig } from '../../config';
 import { logger } from '../../logger';
 import chalk from 'chalk';
 
+const baseRulesets: (Ruleset | ExternalRule)[] = [];
```

Comment:
This will act as an extension and you can still define `--ruleset` or other things (this will be additive)